livro/functions.php

```diff
@@ -63,6 +63,93 @@ function livro_styles() {
 add_action( 'wp_enqueue_scripts', 'livro_styles' );
 
 // Add block patterns
-require get_template_directory() . '/inc/block-patterns.php';
+// require get_template_directory() . '/inc/block-patterns.php';
 
-require get_template_directory() . '/inc/gutenberg-dependency-check.php';
\ No newline at end of file
+// require get_template_directory() . '/inc/gutenberg-dependency-check.php';
```

Comment:
Why is the Gutenberg Dependency check moved from `gutenberg-dependency-check.php` to here?

I would much prefer to keep that logic contained.